frontend/src/scenes/insights/utils/cleanFilters.ts

```diff
@@ -156,11 +158,29 @@ const cleanBreakdownParams = (cleanedParams: Partial<FilterType>, filters: Parti
         if (hasBreakdowns && canMultiPropertyBreakdown) {
             cleanedParams['breakdowns'] = filters.breakdowns
         } else if (hasBreakdowns && isTrends) {
-            cleanedParams['breakdown'] = filters.breakdowns && filters.breakdowns[0].property
-            cleanedParams['breakdown_normalize_url'] = cleanBreakdownNormalizeURL(
-                cleanedParams['breakdown'] as string,
-                filters.breakdown_normalize_url
-            )
+            // Clean up a legacy breakdown
+            if (filters.breakdowns![0].property) {
```

Comment:
```suggestion
            if (filters.breakdowns?.[0].property) {
```

Reply:
I've narrowed the type in the conditions instead of non-null assertions.